docs/node-mixin/config.libsonnet

```diff
@@ -47,6 +47,12 @@
     fsSpaceFillingUpWarningThreshold: 40,
     fsSpaceFillingUpCriticalThreshold: 20,
 
+    // Available disk space (%) thresholds on which to trigger the
+    // 'NodeFilesystemAlmostOutOfSpace' alerts.
+    // 2 thresholds are available : high and low
+    fsOutOfSpaceHigh: 5,
+    fsOutOfSpaceLow: 3,
```

Comment:
To be more consistent with the naming above (L47/48), how about `fsSpaceAvailableWarningThreshold` and `fsSpaceAvailableCriticalThreshold` ?

Reply:
@beorn7 LGTM. I can update the names.

Reply:
@nlamirault could you actually update the names? Then we can merge this.

Reply:
@beorn7 done.